cmd/cfgBuilder/main.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+/*
+Provides the command-line interface for the chainbridge application.
+
+For configuration and CLI commands see the README: https://github.com/ChainSafe/ChainBridge.
+*/
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ChainSafe/ChainBridge/cfgBuilder"
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var app = cli.NewApp()
+
+func init() {
+	app.Action = run
+	app.Copyright = "Copyright 2019 ChainSafe Systems Authors"
+	app.Name = "chainbridge"
+	app.Usage = "ChainBridge V2"
```

Comment:
remove v2 please (would be awesome to remove in cmd/chainbridge too, we dropped this a while ago..)